### DISCUSSION TITLE
Error during rendering markdown

### DISCUSSION THREAD

**Discussion by DemianParkhomenko** [community/community#59636] [Pull Requests]
### Select Topic Area

Bug

## Problem

Github freezes and then shows an error page during the render of the markdown tag.

![image](https://github.com/community/community/assets/95881717/47453720-6b02-4606-9fd0-e7ebe3f1323b)

## How to reproduce

1. Create a Pull Request 
2. Write a description of the Pull Request:

```md
<details><summary>Details</summary>
<p>
https://github.com/<permalink-to-repository-where-PR-created>  
</p>
</details> 
``` 
3. Click 'Preview'. 
![image](https://github.com/community/community/assets/95881717/3d2079a4-99e7-419e-bcde-8ce9a11202f0)
4. Click 'Write'.
<img width="284" alt="image" src="https://github.com/community/community/assets/95881717/cd68868a-c7b7-48df-bd46-03a7f0b8e5f3">

5. Error occurred.
![image](https://github.com/community/community/assets/95881717/d3be1a84-4597-4cf9-9e3a-87d4c60ecafe)

**Comment by leeringrose** [comment]
Hi @DemianParkhomenko ,

I followed your reproduce progress, noticing that on my side it hasn't been happened yet.

I have tried switching tabs many times, but it's never happening. I think it's might be your browser CLI issue.

![Screenshot_1](https://github.com/community/community/assets/132311826/9b4e221e-6692-416f-9d2e-369f95670ded)
![Screenshot_2](https://github.com/community/community/assets/132311826/1e9f059f-c977-4afc-8913-b3ab2dc602c6)

**Reply by DemianParkhomenko** [reply]
It must be a **permalink** that renders like that: 

![image](https://github.com/community/community/assets/95881717/9af79ebe-4f4a-4a54-836a-1e9b073b1d5b)

```md
<details><summary>Details</summary>
<p>

https://github.com/DemianParkhomenko/learn-git/blob/38df17a157953d8d54ddf42553d250af694d69c4/changes.txt#L4-L7

</p>
</details> 
```

But somehow, it works now with exactly the same markdown description 🙂
I apologize for disturbing you.
